api/src/app.ts

```diff
@@ -54,6 +54,9 @@ export default async function createApp(): Promise<express.Application> {
 		process.exit(1);
 	}
 
+	if ((await validateMigrations()) === false || typeof (await validateMigrations()) !== 'boolean') {
```

Comment:
According to the types on `validateMigrations`, this should always return a boolean, so no need to check for the type